usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -26,160 +26,160 @@ The currently available endpoints are listed in the following table.
-|[/api/v2/search/spending_by_category/funding_subagency/](/api/v2/search/spending_by_category/funding_subagency/)|POST| Returns data that is grouped in preset units to support the Spending by Funding Subgency data visualizations on USAspending.gov's Advanced Search page |
-|[/api/v2/search/spending_by_category/naics/](/api/v2/search/spending_by_category/naics/)|POST| Returns data that is grouped in preset units to support the Spending by NAICS data visualizations on USAspending.gov's Advanced Search page |
-|[/api/v2/search/spending_by_category/psc/](/api/v2/search/spending_by_category/psc/)|POST| Returns data that is grouped in preset units to support the Spending by PSC data visualizations on USAspending.gov's Advanced Search page |
-|[/api/v2/search/spending_by_category/recipient_duns/](/api/v2/search/spending_by_category/recipient_duns/)|POST| Returns data that is grouped in preset units to support the Spending by Recipient DUNS data visualizations on USAspending.gov's Advanced Search page |
-|[/api/v2/search/spending_by_category/state_territory/](/api/v2/search/spending_by_category/state_territory/)|POST| Returns data that is grouped in preset units to support the Spending by State Territory data visualizations on USAspending.gov's Recipient Profile page |
-|[/api/v2/search/spending_by_geography/](/api/v2/search/spending_by_geography/)|POST| Returns Spending by state code, county code, or congressional district code |
-|[/api/v2/search/spending_by_transaction/](/api/v2/search/spending_by_transaction/)|POST| Returns awards where a certain subset of fields match against search term |
-|[/api/v2/search/spending_by_transaction_count/](/api/v2/search/spending_by_transaction_count/)|POST| Returns awards where a certain subset of fields match against search term |
-|[/api/v2/search/spending_over_time/](/api/v2/search/spending_over_time/)|POST| Returns transaction aggregated amounts for Spending Over Time data visualizations on USAspending.gov |
-|[/api/v2/search/transaction_spending_summary/](/api/v2/search/transaction_spending_summary/)|POST| Returns the number of transactions and the sum of federal action obligations for prime awards given a set of award of filters |
-|[/api/v2/spending/](/api/v2/spending/)|POST| Returns spending data information through various types and filters |
-|[/api/v2/subawards/](/api/v2/subawards/)|POST| Returns subawards either related, optionally, to a specific parent award, or for all parent awards if desired |
-|[/api/v2/transactions/](/api/v2/transactions/)|POST| Returns transactions related to a specific parent award |
+| Endpoint                                                                                                                                                                   | Methods | Description |
```

Comment:
Not sure if you used a markdown reader to edit this file, but just a word of caution because there are a TON of changes to this file

Reply:
I will confirm my changes; there should not be that many.